packages/vue/package.json

```diff
@@ -28,7 +28,11 @@
     "migrations": "./migrations.json"
   },
   "dependencies": {
-    "tslib": "^2.3.0"
+    "tslib": "^2.3.0",
+    "@nx/devkit": "file:../devkit",
+    "@nx/js": "file:../js",
+    "@nx/linter": "file:../linter",
+    "@nx/vite": "file:../vite"
   },
   "publishConfig": {
     "access": "public"
```

Comment:
Can you please remove the peer dependency on nx? It's supposed to come from `@nx/devkit`.